scripts/generate_visualizer_header.py

```diff
@@ -0,0 +1,35 @@
+# this script generates visualizer header
+import os
+from python_helpers import open_utf8
+
+visualizer_dir = 'extension/visualizer'
+visualizer_css = os.path.join(visualizer_dir, 'visualizer.css')
+visualizer_d3 = os.path.join(visualizer_dir, 'd3.js')
+visualizer_script = os.path.join(visualizer_dir, 'script.js')
+visualizer_header = os.path.join(visualizer_dir, 'include', 'visualizer_constants.hpp')
+
+
+def write_file(filename, varname):
+    with open_utf8(filename, 'r') as f:
+        text = f.read()
+    variable =  'string %s = R"(\n' % (varname,)
```

Comment:
Rather than creating a C++ string, please use the code from the `scripts/generate_csv_header.py` directly which generates a byte array, e.g.:

```py
def get_csv_text(fpath, add_null_terminator = False):
	with open(fpath, 'rb') as f:
		text = bytearray(f.read())
	result_text = ""
	first = True
	for byte in text:
		if first:
			result_text += str(byte)
		else:
			result_text += ", " + str(byte)
		first = False
	if add_null_terminator:
		result_text += ", 0"
	return result_text

```


```cpp
const uint8_t TPCH_QUERIES_q01[] = {83, 69, 76, 69, 67, 84, ...}
```

This avoids any problems with quoting, and avoids having to create a copy of the string on load (which std::string requires).